tensorboard/plugins/core/core_plugin.py

```diff
@@ -89,7 +89,7 @@ def _serve_asset(self, path, request):
       with zipfile.ZipFile(fp) as zip_:
         with zip_.open(path) as file_:
           html = file_.read()
-    return http_util.Respond(request, html, mimetype, expires=3600)
```

Comment:
Isn't this a 3600 second = 60 minute cache?  Re: the correctness of the PR name.

Reply:
+1